config/vite-plugin-yaml.js

```diff
@@ -7,17 +7,20 @@ export default function() {
     name: 'vite-plugin-yaml',
     async transform(code, id) {
       if (yamlExtension.test(id)) {
-        const yamlData = load(code, {
-          filename: id,
-        });
+        try {
+          const yamlData = load(code, {
+            filename: id,
+          });
 
-        return {
-          code: `const data = ${ JSON.stringify(yamlData) };\nexport default data;`,
-          map: { mappings: '' },
-        };
+          return {
+            code: `const data = ${ JSON.stringify(yamlData) };\nexport default data;`,
+            map: { mappings: '' },
+          };
+        } catch (exception) {
+          console.error(`${ id } errored during yaml processing: `, exception);
+          return null;
+        }
       }
-
-      return null;
```

Comment:
Do we need this `return null;` if `(yamlExtension.test(id))` is `false`? I think we are good without it. The handlebars vite plugin doesn't have it: https://github.com/RoundingWell/care-ops-frontend/blob/develop/config/vite-plugin-handlebars-loader.js

Reply:
ooh.. I think it does need it.. we should add that to the handlebars one too

Reply:
Added 👍